e2e/tests/cookie-settings.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { expect, test } from '@playwright/test'
+import { findLast } from 'lodash'
+import vars from '../../src/site/_data/vars'
+
+const gaSessionCookie = `_ga_${vars.googleTagId.replace(/^G-/, '')}`
+
+function transformCookieToObject(cookieString: string) {
+  return Object.fromEntries(
+    cookieString.split('; ').map((cookie) => cookie.split('=')),
+  )
+}
+
+const testOptions = () => {
```

Comment:
I hope this is ok, I wanted to run this description block in multiple contexts but I thought a `forEach`  would have been complicated and hard to read.